.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.14.5
```

Comment:
tests are already being run on 1.15... is there a reason we chose 1.14.5 here?

Reply:
I think that was just a mistake. Not sure why I did that.